Romina.Api/Startup.cs

```diff
@@ -3,14 +3,21 @@
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
+using Romina.Api.Repositories;
 
 namespace Romina.Api
 {
     public class Startup
     {
-        public Startup(IConfiguration configuration)
+        public Startup(IHostingEnvironment env)
         {
-            Configuration = configuration;
+            var builder = new ConfigurationBuilder()
```

Comment:
apologies bud but this is my bad. 

you don't need this here, in the latest version of net core, all the work being done here is handled by the `Program.cs`

so please revert the changes in this constructor. sorry for the confusion